src/components/FilterComponents/poketype/PokeType.js

```diff
@@ -1,3 +1,27 @@
 /**
  * This dropdown filtres different types of pokemons.
- */
\ No newline at end of file
+ */
+ import './PokeType.css';
+ 
+ function PokeType (props) {
+ 
+     function typeSelect(event) {
+ 
```

Comment:
TypeError: props.typeChange is not a function
typeSelect
src/components/FilterComponents/poketype/PokeType.js:10

if i change the dropdown value, i get this error. in HomePage you should add typeChange function to the poketype component